### PR TITLE
[14.0][MIG]account_move_post_block: Migration to 14.0

### PR DESCRIPTION
**Pull request by GuillemCForgeFlow**
Standard migration
@ForgeFlow

### CODE REVIEW

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Review by mariadforgeflow — APPROVED** [review]
LGTM!

**Review by JaumeBforgeFlow — APPROVED** [review]

**Comment by MiquelRForgeFlow** [comment]
@GuillemCForgeFlow if you rebase, the deprecated runbot ci will disappear :)

**Comment by GuillemCForgeFlow** [comment]
> @GuillemCForgeFlow if you rebase, the deprecated runbot ci will disappear :)

Rebased! :+1:

File: account_move_post_block/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "website": "https://github.com/OCA/account-invoicing",
+    "depends": ["account_move_exception"],
+    "data": [
+        "data/account_exception_data.xml",
+        "security/ir.model.access.csv",
+        "security/account_move_post_block_security.xml",
+        "views/account_post_block_reason_view.xml",
+        "views/account_move_view.xml",
+    ],
+    "license": "LGPL-3",
```
Comment (MiquelRForgeFlow) [inline account_move_post_block/__manifest__.py]:
```suggestion
    "license": "AGPL-3",
```

File: account_move_post_block/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 ForgeFlow (http://www.forgeflow.com)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
```
Comment (MiquelRForgeFlow) [inline account_move_post_block/__manifest__.py]:
```suggestion
# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
```
in all the module

Comment (GuillemCForgeFlow) [reply account_move_post_block/__manifest__.py]: Perfect, thank you :)

**Pull request merged**